policyengine_us/variables/gov/states/wv/tax/income/credits/wv_low_income_family_tax_credit.py

```diff
@@ -0,0 +1,48 @@
+from policyengine_us.model_api import *
+
+
+class wv_low_income_family_tax_credit_rate(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "West Virginia low-income family tax credit"
+    unit = "/1"
+    definition_period = YEAR
+    defined_for = "wv_low_income_family_tax_credit_eligible"
+
+    def formula(tax_unit, period, parameters):
+        filing_status = tax_unit("filing_status", period)
+        filing_statuses = filing_status.possible_values
+        p = parameters(
+            period
+        ).gov.states.wv.tax.income.credits.low_income_family_tax_credit
+
+        wv_agi = tax_unit("wv_agi", period)
+        # max family size limit
```

Comment:
make this its own variable for modularity (and to avoid code duplication if we'd kept the separate eligibility variable)